lib/src/viewer.dart

```diff
@@ -225,7 +225,7 @@ class _PDFViewerState extends State<PDFViewer> {
             )
           : null,
       floatingActionButtonLocation: FloatingActionButtonLocation.centerDocked,
-      bottomNavigationBar: (widget.showNavigation || widget.document.count > 1)
+      bottomNavigationBar: widget.showNavigation
```

Comment:
Instead of removing the 2nd part of the condition, we can just change OR to AND.

Reply:
Can you please take a look at this PR @lohanidamodar

Reply:
@Monal5031 I believe your suggestion is better, as we don't want the navigation shown even if the showNavigation is true and page count is just 1. AND instead of removing second condition seems better.